tests/data/group_system/group_auditing/group_auditd_data_retention/rule_auditd_data_retention_space_left/space_left.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_stig-rhel7-disa
+# remediation = bash
+
+if grep -q "^space_left[[:space:]]*= " /etc/audit/auditd.conf; then
+	sed -i "s/^space_left = .*$/space_left = 100/" /etc/audit/auditd.conf
```

Comment:
It seems that you make sure that there is a line `^space_left = ...` and you make a substitution of the whole line to get 100 as right-hand side value. However, the grep regexp has exactly one space between `space_left` and equals. I think that it should be the same as the grep regexp - maybe have it in a variable?

Reply:
I have made the regular expression consistent across tests and remediation.